pubsub/google/cloud/pubsub_v1/subscriber/_protocol/dispatcher.py

```diff
@@ -42,12 +42,8 @@ def start(self):
             if self._thread is not None:
                 raise ValueError("Dispatcher is already running.")
 
-            flow_control = self._manager.flow_control
             worker = helper_threads.QueueCallbackWorker(
-                self._queue,
-                self.dispatch_callback,
-                max_items=flow_control.max_request_batch_size,
-                max_latency=flow_control.max_request_batch_latency,
+                self._queue, self.dispatch_callback, max_items=100, max_latency=0.01
```

Comment:
If these faluse aren't coming from the `flow_control` instance, they should at least be module globals somewhere, so that they can be documented.

Reply:
Agreed, will promote them to module level constants.